cmake/Modules/FindHAPI.cmake

```diff
@@ -11,6 +11,7 @@ GET_FILENAME_COMPONENT(module_file_path ${CMAKE_CURRENT_LIST_FILE} PATH )
 # Look for the header file.
 FIND_PATH(HAPI_INCLUDE_DIR NAMES HAPI/HAPI.h
     PATHS $ENV{H3D_ROOT}/../HAPI/include
+    $ENV{hapi_ROOT}/include
```

Comment:
Why an environment variable and not a CMake variable?
I think it would be more user friendly to have HAPI_ROOT in CMake cache.

Reply:
I know. Here is the problem, I'm using cmake 3.15 and they have a new policy for the `<package>_ROOT` variables:
```
CMP0074
-------

``find_package()`` uses ``<PackageName>_ROOT`` variables.

In CMake 3.12 and above the ``find_package(<PackageName>)`` command now
searches prefixes specified by the ``<PackageName>_ROOT`` CMake
variable and the ``<PackageName>_ROOT`` environment variable.
Package roots are maintained as a stack so nested calls to all ``find_*``
commands inside find modules also search the roots as prefixes.  This policy
provides compatibility with projects that have not been updated to avoid using
``<PackageName>_ROOT`` variables for other purposes.

The ``OLD`` behavior for this policy is to ignore ``<PackageName>_ROOT``
variables.  The ``NEW`` behavior for this policy is to use
``<PackageName>_ROOT`` variables.

This policy was introduced in CMake version 3.12.  CMake version
3.15.0 warns when the policy is not set and uses ``OLD`` behavior.
Use the ``cmake_policy()`` command to set it to ``OLD`` or ``NEW``
explicitly.

.. note::
  The ``OLD`` behavior of a policy is
  ``deprecated by definition``
  and may be removed in a future version of CMake.
```

So if I use `HAPI_ROOT` with cmake default settings it will be ignored.
Is it ok for you if I had to the CMakeLists.txt of HAPI this:
```cmake
if(${CMAKE_VERSION} VERSION_GREATER "3.11.4")
    message(STATUS "policy")
    cmake_policy(SET CMP0074 NEW)
endif()
```
It worked for me and it should stay local to the HAPI module as its CMakeLists is a separate project. What do you think?

Reply:
I agree with your suggestion :+1:

Reply:
the changes are ready, sorry for the delay